third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -1510,16 +1515,9 @@ func resourceContainerClusterDelete(d *schema.ResourceData, meta interface{}) er
 		return err
 	}
 
-	var location string
-	locations := []string{}
```

Comment:
this is a change to resourceContainerClusterDelete unrelated to actual issue - `locations` doesn't get used and getLocation does the same thing to set location